### PR TITLE
Fix bug profile page

### PR DESCRIPTION
**Pull request by khanhnguyen1025**

### CODE REVIEW

File: src/app/pages/welcome/welcome.component.ts [hunk 1]
```diff
@@ -48,26 +46,23 @@ export class WelcomeComponent implements OnInit {
...
+      description: ["", [Validators.required]]
     });
   }
 
   isLogged(): boolean {
     if (this.authService.isLoggedIn()) {
       return true;
     } else {
-      this.router.navigate(['/login']);
+      this.router.navigate(["/login"]);
```
Comment (thanglenova) [inline src/app/pages/welcome/welcome.component.ts]: please keep single quote with string param `'`

File: src/app/pages/welcome/welcome.component.ts [hunk 2]
```diff
@@ -48,26 +46,23 @@ export class WelcomeComponent implements OnInit {
...
 
     this.requestForm = this.formBuilder.group({
-      dayEndOff: ['', [Validators.required]],
-      dayOffType: [0, ''],
-      dayStartOff: ['', [Validators.required]],
-      description: ['', [Validators.required]],
+      dayEndOff: ["", [Validators.required]],
+      dayOffType: [0, ""],
+      dayStartOff: ["", [Validators.required]],
+      description: ["", [Validators.required]]
```
Comment (thanglenova) [inline src/app/pages/welcome/welcome.component.ts]: please keep single quote with string param `'`

File: src/app/pages/welcome/welcome.component.ts [hunk 3]
```diff
@@ -1,23 +1,22 @@
...
+import { AlertService } from "src/app/core/services/alert.service";
+import { DatePipe } from "@angular/common";
 
 @Component({
-  selector: 'app-welcome',
-  templateUrl: './welcome.component.html',
-  styleUrls: ['./welcome.component.css']
+  selector: "app-welcome",
+  templateUrl: "./welcome.component.html",
+  styleUrls: ["./welcome.component.css"]
```
Comment (thanglenova) [inline src/app/pages/welcome/welcome.component.ts]: please keep single quote with string param `'`

File: src/app/pages/welcome/welcome.component.ts [hunk 4]
```diff
@@ -94,42 +88,48 @@ export class WelcomeComponent implements OnInit {
...
-          this.alertService.error('error');
+          this.alertService.error("error");
         }
       );
     this.isVisible = false;
   }
 
   getByYear(year: number): void {
-    const id = +this.route.snapshot.paramMap.get('id');
+    const id = +this.route.snapshot.paramMap.get("id");
```
Comment (thanglenova) [inline src/app/pages/welcome/welcome.component.ts]: please keep single quote with string param `'`

File: src/app/pages/welcome/welcome.component.ts [hunk 5]
```diff
@@ -94,42 +88,48 @@ export class WelcomeComponent implements OnInit {
...
+      .addDayOff(valueForm)
       .pipe(first())
       .subscribe(
         data => {
-          this.alertService.success('successful');
+          this.alertService.success("successful");
         },
         error => {
-          this.alertService.error('error');
+          this.alertService.error("error");
```
Comment (thanglenova) [inline src/app/pages/welcome/welcome.component.ts]: please keep single quote with string param `'`

File: src/app/pages/welcome/welcome.component.ts [hunk 6]
```diff
@@ -94,42 +88,48 @@ export class WelcomeComponent implements OnInit {
...
+      )
     };
-    this.dayOffService.addDayOff(valueForm)
+    this.dayOffService
+      .addDayOff(valueForm)
       .pipe(first())
       .subscribe(
         data => {
-          this.alertService.success('successful');
+          this.alertService.success("successful");
```
Comment (thanglenova) [inline src/app/pages/welcome/welcome.component.ts]: please keep single quote with string param `'`

File: src/app/pages/welcome/welcome.component.ts (hunk 1 above)
Comment (khanhnguyen1025) [reply src/app/pages/welcome/welcome.component.ts]: done

File: src/app/pages/welcome/welcome.component.ts (hunk 2 above)
Comment (khanhnguyen1025) [reply src/app/pages/welcome/welcome.component.ts]: done

File: src/app/pages/welcome/welcome.component.ts (hunk 3 above)
Comment (khanhnguyen1025) [reply src/app/pages/welcome/welcome.component.ts]: done

File: src/app/pages/welcome/welcome.component.ts (hunk 4 above)
Comment (khanhnguyen1025) [reply src/app/pages/welcome/welcome.component.ts]: done

File: src/app/pages/welcome/welcome.component.ts (hunk 5 above)
Comment (khanhnguyen1025) [reply src/app/pages/welcome/welcome.component.ts]: done

File: src/app/pages/welcome/welcome.component.ts (hunk 6 above)
Comment (khanhnguyen1025) [reply src/app/pages/welcome/welcome.component.ts]: done

**Pull request merged**